backend/app/controllers/app.py

```diff
@@ -266,51 +268,6 @@ def generate_for_all_models():
 
         print("Filtered Multiple Models Output", output_filtered)
 
-        temporary_output = [
```

Comment:
Is this the hard coded data being removed?

Reply:
Yea, we had used this initially when the ml_model wasn't set up